app/src/app/initApp.js

```diff
@@ -72,19 +73,27 @@ export default () => {
       }
     )
 
+    // workaround to solve problem:
+    // 1. move grape window to second screen
+    // 2. quit grape -> x or y might be negative now in windw-state.json
+    // 3. unplug monitor
+    // 4. start Grape again
+    // see GRAPE-14546
+    if (state.dimensions.x < 0 || state.dimensions.y < 0) {
+      state.prefs.x = 0;
```

Comment:
are you using this values somewhere? Btw. we don't use semicolons.

Reply:
`new BrowserWindow(state.prefs)` expects those values

Reply:
removed semicolons

Reply:
ok